src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side1;
+
+    public Square(String color, int side1) {
+        super(color);
+        this.side1 = side1;
+    }
+
+    public int getArea() {
+        return side1 * side1;
+    }
+
+    @Override
+    public String getColor() {
+        return color;
+    }
```

Comment:
The 'getColor' method is redundant here because it is inherited from the 'Figure' class and has not been changed. You can remove this override.